packages/appstore/src/components/add-options-account/add-options-account.scss

```
@@ -0,0 +1,26 @@
+.add-options-account {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0.7rem 1.1rem;
+    background: var(--general-section-1);
+    border-radius: 0.5rem;
+    margin: 0 auto;
+    margin-top: 1.6rem;
```

```suggestion
    margin: 1.6rem auto 0;
```
There's some conflicting styles here. Is this what you meant?

---

Yeap 👀